packages/client/src/forms/utils.ts

```diff
@@ -654,8 +655,13 @@ export const convertToMSISDN = (phone: string) => {
       ? 'ZMB'
       : window.config.COUNTRY.toUpperCase()
 
-  const countryCallingCode =
-    callingCountries[countryCode].countryCallingCodes[0]
+  const data = allCountries.find(
+    (countryData) => countryData.iso2 === countryCode.slice(0, 2)
+  )
+
+  const countryCallingCode = data
```

Comment:
I think it would be best if we'd remove the `callingCountries[countryCode].countryCallingCodes[0]` reference altogether and only use the new reference data from this point on. The reason why I think so is that otherwise we will have this ternary statement here that no one remembers the original reason for an we'll be too afraid to break something to remove it